packages/core/src/tracing/instrumentation/protocols/http2Client.js

```diff
@@ -12,10 +12,7 @@ const constants = require('../../constants');
 const httpCommon = require('./_http');
 const readSymbolProperty = require('../../../util/readSymbolProperty');
 const tracingUtil = require('../../tracingUtil');
-const urlUtil = require('../../../util/url');
-
-const discardUrlParameters = urlUtil.discardUrlParameters;
-const filterParams = urlUtil.filterParams;
+const { filterParams, sanitizeUrl } = require('../../../util/url');
```

Comment:
remark: `discardUrlParameters` has been renamed to `sanitizeUrl` because it now discards query parameters and also redacts credentials embedded into the URL.